ui/k-info-panel/switch_info.kytos

```diff
@@ -36,7 +55,9 @@
                   'manufacturer': '',
                   'serial': '',
                   'hardware': '',
-                  'software': ''},
+                  'software': '',
+                  'enabled': '',
+                  'active': '',},
```

Comment:
@Alopalao let's switch the order of both `enabled` and `active` to come before the `id`, that way in the property panel it's always going to start with `enabled` and `active` followed by the rest of the properties in the same way that you've done for the link details on this PR https://github.com/kytos-ng/topology/pull/73/files. As it is, depending on the screen size it'd be at the bottom requiring scrolling, and this is a very important info that operators will look for:

![2022-03-10-171508_417x410_scrot](https://user-images.githubusercontent.com/1010796/157747942-f88058f9-839c-497a-90ad-f54697372286.png)

Reply:
The order of the key has ben changed on the commit  [8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc](https://github.com/kytos-ng/topology/pull/75/commits/8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc)